README.md

```diff
@@ -29,6 +29,10 @@
 > - We've released v0.5.0-beta. You can find release notes [here](https://github.com/linkedin/datahub/releases/tag/v0.5.0-beta)
 > - We're on [Slack](docs/slack.md) now! Ask questions and keep up with the latest announcements.
 
+✨ DataHub participates in Hacktoberfest. Refer https://hacktoberfest.digitalocean.com for more details.
+> - Support open source and pick a limited edition T-shirt or plant a tree.
+> - To earn your Hacktoberfest tee or tree reward, you must register and make four valid pull requests (PRs) between October 1-31 (in any time zone)
```

Comment:
Also give more colors on what "valid PR" means.

Reply:
https://hacktoberfest.digitalocean.com/details#quality 

This contains what a bad PR is. Let me highlight that instead inline with hacktoberfest 😄